product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Step 4 - Installing the PGD software
+navTitle: Installing PGD software
+deepToC: true
+---
+
+## Installing the PGD software
+
+With the repositories configured, you can now install the Postgres Distributed software. 
+These steps must be carried out on each host.
+
+* Install the PGD packages which include a server specific BDR package and generic PGD proxy and cli packages.
+* Initialise the Postgres database.
+* Add the BDR extension to the shared_preload_libraries setting in `postgresql.conf`.
+* Set the `wal_level` GUC variable to `logical` in `postgresql.conf`.
+* Turn on commit timestamp tracking by setting `track_commit_timestamp` to `'on'` in `postgresql.conf`.
+* Raise the maximum worker processes to 16 by setting `max_worker_processes` to `'16'` in `postgresql.conf`.
+* Set a password on the EnterprisedDB/Postgres user.
+* Add rules to `pg_hba.conf` to allow nodes to connect to each other.
+* Add a `.pgpass` file to allow nodes to authenticate each other.
+* Restart the server.
+* Log into the server's default database (`edb` for EPAS, `postgres` for PGE and Community).
+* Use `CREATE DATABASE bdrdb` to create the default PGD replicated database.
+* Log out and then log back in to `bdrdb`.
+* Use `CREATE EXTENSION bdr` to enable the BDR extension and PGD to run on that database.
+
+We will look in detail the steps for EDB Postgres Advanced Server in the worked example below. 
+
+If you are installing PGD with EDB Postgres Extended Server or Community Postgres, the steps are similar, but details such as package names and paths are different. These differences are summarized in [Installing PGD for EDB Postgres Extended Server](#installing-pgd-for-edb-postgres-extended-server) and [Installing PGD for Community Postgresql](#installing-pgd-for-community-postgresql).
+
+## Worked example
+
+### Install the packages
+
+The first step is to install the packages. For each Postgres package, there is a `edb-bdr5-<postgresversion>` package to go with it. 
+For example, if we are installing EDB Postgres Advanced Server (epas) version 16, we would install `edb-bdr5-epas16`. 
+
+There are two other packages to also install: 
+
+- `edb-pgd5-proxy` for PGD Proxy.
+- `edb-pgd5-cli` for the PGD command line tool.
+
+To install all of these packages on a RHEL or RHEL compatible Linux, run:
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+```
+
+### Initializing and starting the database
```

Comment:
These EPAS database initialization steps do not not work at all for ubuntu/debian, they only work for rhel/rocky/alma/sles.  I understand the worked example is for rhel, but what was the actual requirement for the step?  Just ensuring that initdb had been run?  Maybe that could be more clear in the initial bullets...based on these instructions I wasn't sure if I needed to rerun initdb for some reason, since on ubuntu/debian, initdb is automatically run by the edb-asXX-server installer, the enterprisedb user is created automatically, the `edb-as` service is automatically started, and the `edb` database is created.

Reply:
The instructions these are worked from always run initdb/edb-as-<version>

Reply:
so for the worked example on redhat yes that needs to happen because that's how it works on rhel, but the bulleted list of "actions" is platform independent right?   In other words, seems like bullet two "initialize the postgres database" is only required if that wasn't already done by the installer, IIUC, so simply stating "initialize the postgres database (if necessary)" would have made it more clear for me

Reply:
RHEL sets everything up and initializes the database and sets up the users and services too. The instructions are somewhat belt&braces and try to ensure that you're always working with a fresh database, afaict. (ponders)

Reply:
I've tweaked the language now.